drivers/mtd/Kconfig

```diff
@@ -192,6 +192,15 @@ config MTD_CONFIG_NAME_LEN
 	---help---
 		Sets the maximum length of config item names.
 
+config MTD_NVS
+	bool "Enable Dev Non-volatile storage (MTD based) device"
```

Comment:
config MTD_CONFIG_FAIL_SAFE
	bool "Enable Fail Safe"
---help---
		Enable the new storage layout to support the resilient to power loss.